cpp/oneapi/dal/algo/dbscan/backend/gpu/kernel_fp_impl.hpp

```diff
@@ -322,6 +325,7 @@ sycl::event kernels_fp<Float>::update_queue(sycl::queue& queue,
                                             std::int64_t block_start,
                                             std::int64_t block_end,
                                             const bk::event_vector& deps) {
+    ONEDAL_PROFILER_TASK(update_queue, queue);
```

Comment:
I know that it is too late for this comment. But naming `update_queue` is confusing for me....

Reply:
Let's discuss it later